tests/ImageSharp.Tests/project.json

```diff
@@ -20,9 +20,24 @@
     }
```

Comment:
@JimBobSquarePants do you want to StyleCop test projects with the current SC rules? :cry: 

Keep in mind, that the current **actual** test coverage of this project is very low. The ~80% coverage shown by codecov is brought mostly by "integration" tests executing everything on every image without any automatic assertions. These are **not unit tests**. If you remove this cheat from the coverage metric, the results will be very disappointing.

Unit tests should be small, self-documenting pieces of code with a **very low** barrier to write them, to encourage developers to write more and more of them. By forcing to xml-doc everything, and forcing developers to focus on formatting details, you will discourage people to do TDD!

Just look at [corefx test code](https://github.com/dotnet/corefx/blob/master/src/System.Memory/tests/Span/Slice.cs). There are no xml docs at all. And these are microsoft people spending their **job time** to write them. How can you expect people who are spending their free time for this project to do an extra +30% work to formatting & "documenting" test cases? (Well-named, clean AAA test cases are already documented!)

I'm happy to add some comments on non trivial parts, specially if asked for. But making a general overrestricted rule from this, overseen by dumb robots will be a crime against TDD & will act against the goal of having more real test coverage.

TDD is my main methodology to develop software, and I think I'm not alone with this. I **strongly** disagree with this idea :(